modules/xmpp/moderator.js

```diff
@@ -238,13 +242,6 @@ export default class Moderator {
                     .up();
             }
         }
-        if (FeatureFlags.isJoinAsVisitorSupported()) {
```

Comment:
Why we don't need this anymore?

Reply:
Cause we move it in _createConferenceRequest so it will be available for both xmpp and http send method.